ui/src/components/DownloadListItem/index.js

```diff
@@ -2,31 +2,57 @@ import React from 'react';
 import { withPolling } from 'ui/utils/hocs';
 import { compose, withProps } from 'recompose';
 import OptionListItem from 'ui/components/OptionListItem';
+import { activeOrganisationSettingsSelector } from 'ui/redux/modules/auth';
+import { connect } from 'react-redux';
+import moment from 'moment';
+import createdAt from 'ui/containers/Owner/CreatedAt';
 
-const DownloadListItem = ({ model, deleteModel }) => {
-  if (model.get('isReady')) {
+const DownloadListItem = (props) => {
```

Comment:
Why were the parameters changed from `({ model, deleteModel })` to `(props)` when it can be changed to `({ download, deleteDownloadedModel })`? Having it called `props` here makes it look like those `props` are passed in as an object to some child component.